src/userWallet.ts

```diff
@@ -65,6 +65,25 @@ export class UserWallet {
         });
     }
 
+    static loadSecretKey(keyFileObject: any, password: string, addressIndex?: number): UserSecretKey {
```

Comment:
how about simple `decrypt`? it's static in `UserWallet` so you would use it as `UserWallet.decrypt` which sounds straight forward imo

Reply:
Indeed, renamed :+1: